### DISCUSSION TITLE
Input autofill color

### DISCUSSION THREAD

**Discussion by zeledorka** [quasarframework/quasar#14472] [General - Components / Directives / etc]
Hi everyone,

Is there any way to change the color of the autofill on quasar inputs? I found this [thread](https://forum.quasar-framework.org/topic/6099/form-input-autofill-background-color-in-chrome) but none of the suggestions worked for me. Has anyone figured out out how to do this?

Thanks for the help!

**Comment by MilosPaunovic** [comment]
This worked for me on some of the older projects, check if it's still valid:
https://stackoverflow.com/questions/2781549/removing-input-background-colour-for-chrome-autocomplete

**Reply by zeledorka** [reply]
Tried that, didn't work for me

**Comment by pratik227** [comment]
https://codepen.io/Pratik__007/pen/ZEoajPJ

**Comment by MilosPaunovic** [comment]
This works for me on `Quasar@2` and Chrome `Version 105.0.5195.127 (Official Build) (64-bit)` on Windows 10:

```scss

input {
  -webkit-background-clip: text !important;
  background-clip: text !important;
}
```
